packages/amount-selectors/src/components/AmountSelectors.js

```diff
@@ -102,7 +111,24 @@ class AmountSelectors extends React.Component {
   };
 
   render() {
-    const { disabled } = this.props;
+    const {
+      disabled,
+      max,
+      min,
+      minAmountMessage,
+      maxAmountMessage,
+      showBoundariesMessageOnlyOnFocus,
+    } = this.props;
+    const { focused, value } = this.state;
+    const showMessage =
+      !showBoundariesMessageOnlyOnFocus ||
+      (showBoundariesMessageOnlyOnFocus && focused);
+    const showMaxMessage = value > max && showMessage;
+    const showMinMessage = value < min && showMessage;
```

Comment:
When the user deletes the input number, we are showing the `minAmountMessage`, check if the input is empty before show the message.